includes/controls/media.php

```diff
@@ -128,66 +128,141 @@ public function enqueue() {
 	 */
 	public function content_template() {
 		?>
+		<#
+			// For BC.
+			if( data.media_type ) {
+				data.media_types = [ data.media_type ];
+			}
+
+			if( data.should_include_svg_inline_option ) {
+				data.media_types.push( 'svg' );
```

Comment:
Who initialized it if no `data.media_type`?

Reply:
Either the previous condition initialized it, or it's already initialized from the default settings.
There is no case where it gets into this condition without `data.media_types` being initialized.